var/spack/repos/builtin/packages/rmgdft/package.py

```diff
@@ -36,6 +33,8 @@ class Rmgdft(CMakePackage):
 
     variant("local_orbitals", default=True, description="Build O(N) variant.")
 
+    variant("cuda", default=False, description="Build cuda enabled variant.")
```

Comment:
Would it make sense to inherit from `CudaPackage` instead?

Reply:
I've been on vacation so I'm just getting to this. I'm not clear on what you mean by inherit. We often need to explicitly specify whether to build a gpu enabled or cpu only version so automatically building the gpu version if CudaPackage is available is not what we want. Or did you mean something else?

Reply:
> I'm not clear on what you mean by inherit. 

```python
class Rmgdft(CMakePackage, CudaPackage):
    ...
```

We have a mixin class called `CudaPackage` that takes care of common operations of cuda packages, like declaring a `cuda` variant etc. It also encodes in one place information on `cuda`

Reply:
I changed it to use CudaPackage as suggested and also added v5.4.0 since that was released after the PR was created.

Reply:
This can be removed, since now it's inherited from the `CudaPackage`:
```suggestion
```